RUMM-1206 Make `DatadogCrashReporting` static in `Package.swift` (#449)
DataDog/dd-sdk-ios: dependency-manager-tests/spm/README.md

```bash
$ make
```

> This will create the `SPMProject.xcodeproj` from `SPMProject.xcodeproj.src` by replacing the `REMOTE_GIT_BRANCH` string with the name of the remote branch.

Reviewer: ```suggestion
This creates the `SPMProject.xcodeproj` from `SPMProject.xcodeproj.src` by replacing the `REMOTE_GIT_BRANCH` string with the name of the remote branch.
```

> Launch the `SPMProject.xcodeproj` to check if SPM is able to fetch `Datadog` package from remote and build the project.

Reviewer: ```suggestion
Launch the `SPMProject.xcodeproj` to check if SPM is able to fetch the `Datadog` package from remote and build the project.
```

> If you need to update the setup in `SPMProject.xcodeproj`, use:

Reviewer: ```suggestion
To update the setup in `SPMProject.xcodeproj`, use:
```

```bash
make create-src-from-xcodeproj
```

> to apply it back to the `SPMProject.xcodeproj.src`, so it can be committed to git.

Reviewer: ```suggestion
to apply the change back to the `SPMProject.xcodeproj.src`, so it can be committed to git.
```
